dts/bindings/xen,domu.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xen DomU configuration
+compatible: "xen,domu"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+    description: Domain ID
```

Comment:
Domain ID is generally assigned automatically.

Reply:
I'll remake it